src/Narochno.Slack/SlackClient.cs

```diff
@@ -43,13 +46,68 @@ public SlackClient(HttpClient httpClient, SlackConfig slackConfig)
             message.Channel = message.Channel.Fallback(slackConfig.Channel);
             message.Emoji = message.Emoji.Fallback(slackConfig.Emoji);
 
-            var json = JsonConvert.SerializeObject(message, serializerSettings);
+            string json = JsonConvert.SerializeObject(message, serializerSettings);
 
-            var response = await GetRetryPolicy().ExecuteAsync(() => httpClient.PostAsync(slackConfig.WebHookUrl, new StringContent(json, Encoding.UTF8, "application/json"), ctx));
+            HttpResponseMessage response = await GetRetryPolicy().ExecuteAsync(() => httpClient.PostAsync(slackConfig.WebHookUrl, new StringContent(json, Encoding.UTF8, "application/json"), ctx));

```

Comment:
I have stopped using `var` in C# and `auto` in C++ except in a few circumstances. It started when I was writing code for Estranged, which I follow the Epic coding style for, and they discourage `auto` with I think some good reasoning: https://docs.unrealengine.com/latest/INT/Programming/Development/CodingStandard/#the'auto'keyword

In C# I've taken to using `var` only in the following places, I think by default ReSharper enforces these too:
- If you are declaring the type - `var local = new Thing();`
- If the type is obvious to the reader - `var local = JsonConvert.DeserializeObject<Thing>(json);`
- If the type has a stupid number of generic arguments
- Dynamic/expando objects ... well, no choice here

Anywhere else I think it makes the code more readable if you just use the type. I'd like to do this for Narochno stuff too... but ofc if you hate it I don't have to.